### DISCUSSION TITLE
ACM Blueprints restore permalinks structure

### DISCUSSION THREAD

**Discussion by blakewilson** [wpengine/atlas-content-modeler#461] [Ideas]
Faust.js WordPress sites usually rely on a custom permalink structure to ensure the WP links are the same as the Next.js file system based routing. it would be great if ACM blueprints could support import/export of WordPress options [as noted here](https://github.com/wpengine/atlas-content-modeler/blob/main/docs/blueprints/index.md#not-yet-supported), to avoid the step of having users setup permalinks manually.

**Comment by nickcernis** [comment]
ACM 0.15.0 is out now and supports this feature via a --wp-options flag in `wp acm export blueprint`.

Updated docs at https://github.com/wpengine/atlas-content-modeler/blob/main/docs/wp-cli/index.md#wp-acm-blueprint-export

Short version: pass the options you want to export to --wp-options. For example: 

```
wp acm blueprint export --wp-options='blogname, permalink_structure'
```

`wp acm blueprint import` will then automatically set any `wp-options` in the `acm.json` file.
